### PR TITLE
feat: added more fields to the metrics

### PR DESCRIPTION
**Pull request by caarlos0**
namely:

- license
- language
- if it is a fork
- if it is archived

PS: maybe it doesn't even make sense to report metrics on archived repositories, as they can't be changed... if you agree, I can fix that in this PR as well :)

Thanks!

### CODE REVIEW

**Comment by caarlos0** [comment]
cc/ @Rucknar

**Comment by Rucknar** [comment]
Thanks for the PR! Code looks good to me, could you also update the docs? I've listed the metrics people can expect in the `metrics.md` file in the root of the repo.

**Comment by caarlos0** [comment]
hi @Rucknar 

I wasn't sure about what was the best way of doing that, so I basically re-generated that output using "live" data. Let me know what you think!

Cheers!

File: exporter/metrics.go [hunk 1]
```diff
@@ -11,27 +11,27 @@ func AddMetrics() map[string]*prometheus.Desc {
...
 		prometheus.BuildFQName("github", "repo", "open_issues"),
 		"Total number of open issues for given repository",
-		[]string{"repo", "user", "private"}, nil,
+		[]string{"repo", "user", "private", "fork", "arhived", "license", "language"}, nil,
 	)
 	APIMetrics["Watchers"] = prometheus.NewDesc(
 		prometheus.BuildFQName("github", "repo", "watchers"),
 		"Total number of watchers/subscribers for given repository",
-		[]string{"repo", "user", "private"}, nil,
+		[]string{"repo", "user", "private", "fork", "arhived", "license", "language"}, nil,
```
Comment (tianon) [inline exporter/metrics.go]: Shouldn't `arhived` in all these rather be `archived`? (minor typo)

Comment (caarlos0) [reply exporter/metrics.go]: good catch, fixed, thanks!

**Pull request merged**